configuration/configuration_blackbox_test.go

```diff
@@ -31,7 +40,7 @@ func TestMain(m *testing.M) {
 
 func resetConfiguration() {
 	var err error
-	config, err = configuration.GetConfigurationData()
+	config, err = configuration.NewConfigurationData("../config.yaml")
```

Comment:
@alexeykazakov , 

The order of resolution of config values is 

1. check defaults in the code.
2. check if path is provider `config.yaml` , override `1` if yes.
3. check if env vars is present, override `2` if yes.

Now, the values in (1) config.yaml ( present in repo ) executed during tests are exactly the same as the (2) defaults declared in the `configuration.go' . So, whether or not we pass the path hardly makes any difference in tests.
( We conventionally try to enforce whenever a new config var is added, add defaults to both code and yaml )

As for env variables, they are already handled inside the tests ( for existing ones, not in this PR though ).